src/src_V2/modules/manager/supplier-management/activity/dialogs/Index.jsx

```diff
@@ -0,0 +1,26 @@
+import { Dialog } from '@mui/material';
+import ActivitiesDetailsDialog from './ActivityDetails/ActivitiesDetails.dialog';
+
+function CurrentDialog({ data, onClose }) {
+  switch (data.mode) {
+    case 'activityDetails':
+      return <ActivitiesDetailsDialog data={data} onClose={onClose} />;
+    default:
+      return null;
+  }
+}
+
+export default function DialogManager({ data, onShowHideDialog }) {
+  function onClose() {
+    onShowHideDialog({
+      mode: '',
+      open: false,
+    });
+  }
+
+  return (
+    <Dialog open={data.open} onClose={onClose} fullWidth PaperProps={{ width: 750 }}>
```

Comment:
nned we set this `fullWidth PaperProps={{ width: 750 }}` ?

Reply:
removed it